src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,206 @@
+  --pf-c-inline-edit__group--m-footer--MarginTop: var(--pf-global--spacer--xl);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  .pf-m-display-inline-flex {
+    display: inline-flex;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+// Group
+.pf-c-inline-edit__group {
+  display: flex;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__group--item--MarginRight);
+  }
+
+  // Value group
+  &.pf-m-value-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-value-group--item--MarginRight);
+  }
+
+
+  // Input group
+  &.pf-m-input-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--item--MarginRight);
+
+    .pf-m-action-group {
+      --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-c-inline-edit__group--m-input-group--m-action-group--item--MarginRight);
+      --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+    }
+  }
+
+  // Action group
+  &.pf-m-action-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-action-group--item--MarginRight);
+  }
+
+  &.pf-m-input-group,
+  &.pf-m-action-group {
+    display: none;
+  }
+
+  // Icon group
+  &.pf-m-icon-group {
+    --pf-c-inline-edit__group--item--MarginRight: var(--pf-c-inline-edit__group--m-icon-group--item--MarginRight);
+  }
+
+  // Inline group
+  &.pf-m-inline {
+    --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+  }
+
+  // Footer
+  &.pf-m-footer {
+    margin-top: var(--pf-c-inline-edit__group--m-footer--MarginTop);
+  }
+
+  // Column
+  &.pf-m-column {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+
+    flex-direction: column;
+  }
+
+  // Enable groups
+  &.pf-m-enable {
+    display: inline-flex;
+
+    .pf-c-inline-edit__action {
+      display: inline-block;
+    }
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+  }
+}
+
+// Table action cells and action groups inside input groups
+.pf-c-inline-edit .pf-c-table__action,
+.pf-c-inline-edit__group.pf-m-input-group {
+  .pf-c-inline-edit__group.pf-m-action-group {
+    --pf-c-inline-edit__group--m-action-group--item--MarginRight: 0;
+    --pf-c-inline-edit__group--m-action-group--MarginTop: 0;
+  }
+}
+
+// Input and input group
+.pf-c-inline-edit__input,
+.pf-c-inline-edit__group.pf-m-input-group {
+  display: none;
+  flex: 1;
+  visibility: hidden;
+}
+
+// Action
+.pf-c-inline-edit__action {
+  display: none;
+
+  &.pf-m-enable {
+    display: inline-block;
+  }
+
+  &.pf-m-valid .pf-c-button.pf-m-plain {
+    --pf-c-button--m-plain--Color: var(--pf-c-inline-edit__action--m-valid--c-button--m-plain--Color);
+
+    &:hover {
```

Comment:
this rule doesn't appear to be necessary since the rule above overriding `--pf-c-button--m-plain--Color` has higher specificity than the button components declaration to change `--pf-c-button--m-plain--Color: var(--pf-c-button--m-plain--hover--Color);`. Also if you wanted to style the hover color anyways, I believe you could just define `--pf-c-button--m-plain--hover--Color: var(--pf-c-inline-edit__action--m-valid--c-button--m-plain--hover--Color);` in the parent selector - no need to put it in a `&:hover` block